use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -107,24 +114,29 @@ static unsigned int   NumActiveClients    = 0;
  * Simple argument parsing structure.
  */
 struct option Long_options[] = {
-      { "help"                      , no_argument         , NULL, 'h'}
+      { "clock-default"             , no_argument         , NULL, 'd'}
+    , { "help"                      , no_argument         , NULL, 'h'}
     , { "clock-monotonic"           , no_argument         , NULL, 'm'}
     , { "clock-process-cputime-id"  , no_argument         , NULL, 'p'}
     , { "clock-realtime"            , no_argument         , NULL, 'r'}
     , { "clock-thread-cputime-id"   , no_argument         , NULL, 't'}
     , { NULL, 0, NULL, 0}           // End of options
 };
 
-const char * Options_str = ":hrmptd";
+const char * Options_str = ":dhmprt";
```

Comment:
Rearranged to match the order in which options are listed in above lookup array.